src/simplewallet/simplewallet.cpp

```diff
@@ -2057,6 +2057,9 @@ simple_wallet::simple_wallet()
   m_cmd_binder.set_handler("sweep_unmixable",
                            boost::bind(&simple_wallet::sweep_unmixable, this, _1),
                            tr("Send all unmixable outputs to yourself with ring_size 1"));
+  m_cmd_binder.set_handler("sweep_mixable",
+                           boost::bind(&simple_wallet::sweep_mixable, this, _1),
+                           tr("Send all mixable outputs to yourself with ring_size 1"));
```

Comment:
That should not be possible. If it is, then it is a bug.
Otherwise, if you make sure ring size is correct, seems OK.